server/controllers/airlines.js

```diff
@@ -0,0 +1,19 @@
+const Airline = require('../models').Airline;
```

Comment:
Please use the destructuring assignment syntax here, as we want to avoid redundant code and keep it nice and clean
```
const { Airline } = require('../models');
```

Please update this in the other files as well.